src/GitHub/Data/Issues.hs

```diff
@@ -36,6 +36,7 @@ data Issue = Issue
     , issueId          :: !(Id Issue)
     , issueComments    :: !Int
     , issueMilestone   :: !(Maybe Milestone)
+    , issueReactions   :: !(Maybe IssueReactions)
```

Comment:
I suppose this is https://docs.github.com/en/rest/reactions#list-reactions-for-an-issue , is it?
Would you mind adding the link as a haddock comment to this field?